docs/glossary.md

```diff
@@ -38,4 +38,9 @@ Kerberos is an authentication protocol for client/server authentication without
 
 ## Rolling restart
 
-A rolling restart (rolling upgrade) is shutting down and upgrading nodes one by one. The whole cluster remains operational. There is no interruption to clients assuming the elections are short and all writes directed to the old primary use the retryWrite mechanism.
\ No newline at end of file
+A rolling restart (rolling upgrade) is shutting down and upgrading nodes one by one. The whole cluster remains operational. There is no interruption to clients assuming the elections are short and all writes directed to the old primary use the retryWrite mechanism.
+
+## Technical preview feature
+
+Technical preview features are not yet ready for enterprise use and are not included in support via SLA. They are included in this release so that users can provide feedback prior to the full release of the feature in a future GA release (or removal of the feature if it is deemed not useful). This functionality can change (APIs, CLIs, etc.) from tech preview to GA.
```

Comment:
Is "enterprise use" meaning the same as "production use"? I am curious if we can use "production use" here.

Reply:
Can we somehow mention that potential users can test or "hands on" technical preview feature before providing ffedback?